### PR TITLE
can't read from cache if cache doesn't exist

### PR DESCRIPTION
**Pull request by octatone**
OMG THIS HURTS

### CODE REVIEW

File: public/lib/events.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ var events = {
 
       // if this was a published event, do an immediate trigger
       var cache = self._cache;
-      if (cache[name]) {
+      if (cache && cache[name]) {
```
Comment (netroy) [inline public/lib/events.js]: but cache is created in the constructor.. even though this might be a fix, it's definitely not here where the problem is.

Comment (octatone) [inline public/lib/events.js]: I think this is happening in the case where an object has been destroyed or is being destroyed?

Comment (octatone) [inline public/lib/events.js]: This is the trace I am looking at: https://bugsnag.com/6wunderkinder-1/webapp/errors/5509d8607c992b99eacff166#comments

**Comment by octatone** [comment]
Invalid

**Pull request closed without merging**